ai2thor_wrapper/config.ini

```diff
@@ -0,0 +1,5 @@
+[ENV_SPECIFIC]
+;PICKUPABLE_OBJECTS=Mug
+PICKUPABLE_OBJECTS=Mug,Apple,Book,Bowl,Plate,Tomato,Pan,Fork,Knife,Spoon,Potato,Lettuce,Egg,ButterKnife
+ACCEPTABLE_RECEPTACLES=CounterTop,TableTop,Sink
```

Comment:
I would rename those two to these:
PICKUP_OBJECTS. It sounds more natural to just say pick up.
RECEPTACLE_OBJECTS. Also a property of a specific group of objects, we can use objects in the name as convention

Reply:
This is the only point in my life I can rejoice in finally being able to call something ACCEPTABLE_RECEPTACLES and you want it renamed?

PICKUP_OBJECTS sure